src/garage/torch/utils.py

```diff
@@ -2,77 +2,233 @@
 import torch
 
 
-def np_to_torch(array_dict):
-    """Convert numpy arrays to PyTorch tensors.
+class Utils:
+    """A collection of common functions that are used by Pytorch algos.
 
-    Args:
-        array_dict (dict): Dictionary of data in numpy arrays.
-
-    Returns:
-        dict: Dictionary of data in PyTorch tensors.
+    This collection of functions can be used to manage the following:
+        - Pytorch GPU usage
+            - setting the default Pytorch GPU
+            - converting Tensors to GPU Tensors
+            - Converting Tensors into `numpy.ndarray` format and vice versa
+        - Updating model parameters
 
     """
-    for key, value in array_dict.items():
-        array_dict[key] = torch.FloatTensor(value)
-    return array_dict
-
-
-def torch_to_np(value_in):
-    """Convert PyTorch tensors to numpy arrays.
-
-    Args:
-        value_in (tuple): Tuple of data in PyTorch tensors.
+    use_gpu = False
```

Comment:
These are basically still just globals. I think the file was better as it was before, since the import pattern was more obvious. This just feels like trying to emulate Java in Python.

Of course, that requires telling pylint to ignore your globals, but I think that's fine.

Reply:
But the problem with this is that we have these variables in the global scope in the older version. While it be very unlikely for users to name their variables something like _USE_GPU, I feel that this is still a bad programming practice. 

Would you like me to change back to the older versioning of things with the globals? @krzentner

Reply:
Python "globals" are still affected by namespaces. It's only "global" to this module.

Reply:
So that isn't actually a problem. I think it was clearer in the older version, since these are actually acting like globals.